style.css

```diff
@@ -0,0 +1,115 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 16px 16px;
+  background-color: #f7f7f9;
+}
+
+.background-img {
+  background-image: url(./img/mobile-background.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 100%;
+}
```

Comment:
- [ ] as in the previous review please add the border-radius in the bottom left side of the headline section

**_Hint : you can add `border-bottom-left-radius: 50px;` to your section styling_**



![image](https://user-images.githubusercontent.com/89282221/192023125-ef740ae2-415a-4ea1-a29f-b8a2893f06a0.png)